sdk/inc/azure/iot/az_iot_provisioning_client.h

```diff
@@ -329,7 +331,7 @@ AZ_NODISCARD az_result az_iot_provisioning_client_register_get_publish_topic(
  * @remark The payload of the MQTT publish message should be empty.
  *
  * @param[in] client The #az_iot_provisioning_client to use for this call.
- * @param[in] register_response The received #az_iot_provisioning_client_register_response response.
+ * @param[in] operation_id The received #az_iot_provisioning_client_register_response response.

```

Comment:
Doc comment needs update.
```suggestion
 * @param[in] operation_id The received operation id from the #az_iot_provisioning_client_register_response response.
=
```

Reply:
was already fixed in the PR

Reply:
I meant the actual **description** of the parameter documentation. See suggested change.